src/hooks/usePipelineRunsForApplication.ts

```diff
@@ -3,10 +3,24 @@ import { useK8sWatchResource } from '@openshift/dynamic-plugin-sdk-utils';
 import { PipelineRunLabel } from '../consts/pipelinerun';
 import { ComponentGroupVersionKind } from '../models';
 import { PipelineRunGroupVersionKind } from '../shared';
-import { PipelineRunKind } from '../shared/components/pipeline-run-logs/types';
-import { ComponentKind } from '../types';
+import { ComponentKind, PipelineRunKind } from '../types';
 import { useWorkspaceInfo } from '../utils/workspace-context-utils';
 
+export const usePipelineRun = (pipelineRunName: string): [PipelineRunKind, boolean, unknown] => {
+  const { namespace } = useWorkspaceInfo();
+  const [pipelineRun, loaded, error] = useK8sWatchResource<PipelineRunKind>({
+    groupVersionKind: PipelineRunGroupVersionKind,
+    name: pipelineRunName,
+    namespace,
```

Comment:
I prefer if this hook relied on the namespace being passed in so it's more reusable in cases where later we have access to the namespace already.

It also makes the api more consistent with the other hooks.